docs/template.md

```diff
@@ -16,28 +16,124 @@ Each template file must be set up in the following format:
-- **axiom annotations**: ROBOT can also annotate logical and annotation axioms.
-    - `>A` annotation on annotation: Annotates the annotation axiom created from the cell to the left with the cell value. The column to the left must be an `A*` template string.
-    - `>C` annotation on class expression: Annotates the class expression axiom created from the cell to the left with the cell value. The column to the left must be a `C` template string.
-
+- `>A` (**axiom annotations**): ROBOT can also annotate logical and annotation axioms. The axiom annotation will be on the axiom created on the cell to the left of the `>A*` template string. The `>` symbol can be used in front of any valid annotation character (`>A`, `>AT`, `>AL`, `>AI`)
+   
 Sometimes you want to include zero or more values in a single spreadsheet cell, for example when you want to allow for multiple annotations or have seperate logical axioms. If a template string also contains `SPLIT=|`, then ROBOT will use the `|` character to split the contents of a cell in that column and add an annotation for each result (if there are any). Instead of `|` you can specify a string of characters of your choice - other than pure whitespace - to split on (e.g. `SPLIT=, `).
 
+### Class Template Strings
+
+- `CLASS_TYPE`: ROBOT creates a class for each row of data with a that has a `TYPE` of `class` or `owl:Class`. The class type can be ONLY ONE OF:
+    - `subclass`: the created class will be asserted to be a subclass of each templated class expression (default)
+    - `disjoint`: the created class will be disjoint from each templated class expression, meaning the classes cannot share subclasses
+    - `equivalent`: the created class will be asserted to be equivalent to the intersection of all the templated class expressions
+- `C` **class expression**: If the template string starts with a `C` and a space then it will be interpreted as a class expression. The value of the current cell will be substituted into the template, replacing all occurrences of the percent `%` character. Then the result will be parsed into an OWL class expression. 
+    - ROBOT uses the same syntax for class expressions as Protégé: [Manchester Syntax](http://www.w3.org/2007/OWL/wiki/ManchesterSyntax). This means that an entity can be referred to by its `rdfs:label` (enclosing in single quotes if it has a space in it). 
+    - If it does not recognize a label, ROBOT will assume that you're trying to refer to a class by its IRI (or compact IRI). This can lead to unexpected behaviour, but it allows you to refer to classes (by IRI) without loading them into the input ontology. This is particularly useful when the input ontology would be too large, such as the NCBI Taxonomy.
+    - Properties in class expression **must** be referred to by label in order to be parsed.
+
+#### Example of Class Template Strings
+
+| Label | Entity Type | Class Type | Related classes | Other axioms |
+| --- | --- | --- | --- | --- |
```

Comment:
It's a bit of a pain, but please use spaces to align these columns so the table reads better in raw Markdown.